fbc-integration-test/catalog-source/release/managed-workspace/release_strategy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: fbc-strategy
+  namespace: managed-release-team-tenant
+spec:
+  pipeline: fbc-release
+  bundle: quay.io/scoheb/pipeline-fbc-release:latest
+  policy: fbc-policy
+  serviceAccount: release-service-account
+  params:
+    - name: fromIndex
+      ## staging
+      # value: "quay.io/rh-osbs/iib-pub-pending:v4.13"
+      ## prod
+      # value: "quay.io/rh-osbs/iib-pub:v4.13"
```

Comment:
you can remove the commented lines. We have the prod and preview documented in another repo.